buildbot.tac

```diff
@@ -23,7 +26,3 @@ from flocker_bb.upgrade import UpgradeService
 
 master = BuildMaster(basedir, configfile)
 UpgradeService(basedir, configfile, master).setServiceParent(application)
-
-
-from twisted.application.internet import TimerService
-TimerService(30, master.botmaster.maybeStartBuildsForAllBuilders).setServiceParent(master)
```

Comment:
Ah.  `EC2BuildSlave` now polls for builds to start by setting up a `TimerService` in its `__init__`.